cloudinit/net/eni.py

```diff
@@ -308,18 +308,6 @@ def _parse_deb_config_data(ifaces, contents, src_dir, src_path):
             ifaces[iface]["auto"] = False
 
 
-def parse_deb_config(path):
```

Comment:
parse_deb_config is used by _ifaces_to_net_config_data function several lines below...

Reply:
Doh! I didn't spot that line was inside a comment block.

Reply:
All good. Updating the comments is important too :)